gizmos.js

```diff
@@ -411,6 +411,8 @@ function (dojo, declare) {
 						this.addActionButton( 'button_file', _('File'), 'fileSelectedCard' );
 						this.addActionButton( 'button_cancel', _('Cancel'), 'cancel' );
 						dojo.addClass( 'button_file', 'disabled');
+						dojo.query('.selectable').removeClass('selectable');
+						dojo.query('.row_card').addClass('selectable');
```

Comment:
I have no confidence that a rollback is needed or not in other triggered actions 🤔